compiler/optimizer/LoopCanonicalizer.cpp

```diff
@@ -2000,6 +2004,28 @@ void TR_LoopCanonicalizer::eliminateRedundantInductionVariablesFromLoop(TR_Regio
                   }
                }
 
+            TR::Node *node;
+            ListIterator<TR::Block> blockIt(&blocksInNaturalLoop);
+            TR::Block * derivedInductionVarBlock;
```

Comment:
This variable is not used outside the `for` loop. Can we move the definition there?